File: JAVA/1436. Destination City.java
import java.util.*;

class Solution {
    public String destCity(List<List<String>> p) {
        Map<String, Integer> m = new HashMap<>();
        for (List<String>i : p) {
            m.put(i.get(0), m.getOrDefault(i.get(0), 0) + 1);
            m.putIfAbsent(i.get(1), 0);
        }
        for (Map.Entry<String, Integer> entry : m.entrySet()) {
            if (entry.getValue() == 0)
                return entry.getKey();
        }
        return "";
    }
}

File: C++/1436. Destination City.cpp
class Solution {
public:
    string destCity(vector<vector<string>>& p) {
        map<string,int>m;
        for(int i=0;i<p.size();i++){
            m[p[i][0]]++;
            m[p[i][1]]+=0;
        }
        for(auto it:m){
            if(it.second==0)
            return it.first;
        }
        return " ";
    }
};